packages/bff/src/model/domain/errors.ts

```diff
@@ -6,12 +6,36 @@ import {
 
 export const errorCodes = {
   purposeNotFound: "0001",
+  userNotFound: "0001",
+  selfcareEntityNotFilled: "0032",
```

Comment:
we should reorganize the errorCodes. In this case, you can use 0002 for userNotFound and 0003 for selfcareEntityNotFilled.

Reply:
I followed exactly the scala code, do i have to change the error codes?

Reply:
yes! it's not important to have the same scala error codes. We could redefine the error codes starting from 0001